docs/source/configuration/overview.mdx

```diff
@@ -742,6 +742,96 @@ query AliasedQuery {
 }
 ```
 
+<MinVersion version="1.50.0">
+
+### Extended reference reporting
+
+</MinVersion>
+
+<ExperimentalFeature />
+
+<EnterpriseFeature linkWithAnchor="https://www.apollographql.com/pricing#graphos-router" />
+
+Beginning in v1.50.0, you can configure the router to report enum and input object references for enhanced insights and operation checks.
+Apollo's legacy reference reporting doesn't include data about enum values and input object fields, meaning you can't view enum and input object field usage in GraphOS Studio.
+Legacy reporting can also cause [inaccurate operation checks](#enhanced-operation-checks).
+
+Configure extended reference reporting in `router.yaml` with the `telemetry.apollo.experimental_apollo_metrics_reference_mode` option like so:
+
+```yaml title="router.yaml"
+telemetry: 
+  apollo:
+    experimental_apollo_metrics_reference_mode: extended # Default is legacy
+```
+#### Configuration effect timing
+
+Once you configure extended reference reporting, you can view enum value and input field usage alongside object [field usage in GraphOS Studio](/graphos/metrics/field-usage) for all subsequent operations.
+
+Configuring extended reference reporting automatically turns on [enhanced operation checks](#enhanced-operation-checks), though you won't see an immediate change in your operations check behavior.
+
+This delay is because operation checks rely on historical operation data.
+To ensure sufficient data to distinguish between genuinely unused values and those simply not reported in legacy data, enhanced checks require some historical operations with extended reference reporting turned on.
+
+#### Enhanced operation checks
+
+Thanks to extended reference reporting, operation checks can more accurately flag issues for changes to enum values and input object fields. See the comparison table below for differences between standard operation checks based on legacy reference reporting and enhanced checks based on extended reference reporting.
+
+<table class="field-table api-ref">
+<tr>
+<th style="min-width: 100px;"></th>
+<th style="min-width: 200px;">
+Standard Check Behavior<br/>
+(Legacy reference reporting)
+</th>
+<th>
+Enhanced Check Behavior<br/>
+(Extended reference reporting)
+</th>
+</tr>
+<tr>
+<td>
+
+##### Enum value removal
+
+</td>
+<td>Removing any enum values is considered a breaking change if any operations use the enum.</td>
+<td>Removing enum values is only a breaking change if historical operations use the specific enum value(s) that were removed.</td>
+</tr>
+<tr>
+<td>
+
+##### Default argument changes for input object fields
+
+</td>
+
+<td>Changing or removing a default argument is generally considered a breaking change, but changing or removing default values for input object fields isn't.</td>
+<td>Changing or removing default values for input object fields is considered a breaking change.</td>
```

Comment:
Can we maybe add some information here about how there is a checks config option to ignore default value changes (not removals)? See [this slide](https://docs.google.com/presentation/d/19waRBNbd_-62r9uHLg4AHtQPNz34fmH215TKcxIhR7k/edit#slide=id.g28f84195726_0_28) for the UI - it's at the bottom of the checks config page.

Reply:
Good call! Updated here: https://github.com/apollographql/router/pull/5513/commits/a8d9215c4cefc973ae9b11f97c92e45f54e2d9f1

I'll update the [ignored condition settings docs](https://www.apollographql.com/docs/graphos/delivery/schema-checks/#ignored-conditions-settings) to call out the "not removals" bit.